llvm/test/CodeGen/AArch64/GlobalISel/translate-sve-formal-argument.ll

```diff
@@ -0,0 +1,389 @@
+; NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=aarch64-linux-gnu -O0 -mattr=+sve -global-isel -global-isel-abort=1 -aarch64-enable-gisel-sve=1 \
+; RUN:     -stop-after=irtranslator -verify-machineinstrs %s -o - | FileCheck %s
+
+;; vscale x 128-bit
```

Comment:
Could you add tests with several legal arguments to show the correct usage of the z registers?

Reply:
Isn't [sve-formal-argument.ll](https://github.com/llvm/llvm-project/pull/95236/files#diff-e38af48ccacbc706e49c732d40e671d7b38f6296861cb2947728bd95e8b56d8c) doing this?

Reply:
Yeah. Maybe. But it is assembler.

Reply:
Added more tests